core/src/term/mod.rs

```diff
@@ -116,8 +116,12 @@ pub enum Term {
     #[serde(skip)]
     Var(LocIdent),
 
-    /// An enum variant.
+    /// An enum tag, or equivalently, an enum variant without any argument.
     Enum(LocIdent),
+    /// An applied enum variant (an algebraic data type). In Nickel ADTs can have at most one
```

Comment:
If `EnumVariant` will be treated separately from `Enum`, it's *exactly* one variant, right?

Reply:
I've reworded this to be hopefully a bit clearer (it wasn't entirely clear if a bare enum tag is considered an ADT or not; I think it is, so I've reflected that in the latest version)